### PR TITLE
Issue # 20 and #22

### LINKED ISSUE #20: Ion Pumps use own pressure reading after turning on
Ion Pumps should use their own pressure readings for validating setpoints and hysteresis after they have been turned on through the pressure of another gauge like an pirani gauge or a cold cathode.

### LINKED ISSUE #22: Update MKS 909 Hot Cathode POU with logging and all other upgrades
Since TMO plans to use this MKS hot cathode, please update the POU to include logging and any other upgrades that happened recently.

### PR DESCRIPTION
**Pull request by ghalym**
Issue#20 : Ion Pumps use own pressure reading after turning on, also added a method for AutoOn. user program to invoke the method. Also available through pragma.
Issue#22, Update MKS 909 Hot Cathode POU with logging and all other upgrades.

### CODE REVIEW

**Comment by slacAWallace** [comment]
Maggie, you should use a more descriptive title and include the issues in the text so they automatically link.

File: L2SIVacuum/POUs/Functions/Gauges/FB_MKS909.TcPOU [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+
+
+//STATE Logger  
+IF ePrevState <> IG.eState THEN
+	  CASE IG.eState OF
+		ValidHi:
+		ValidLo:
+		Valid:
+			fbLogger(sMsg:='Gauge pressure is in valid state.', eSevr:=TcEventSeverity.Critical);
```
Comment (slacAWallace) [inline L2SIVacuum/POUs/Functions/Gauges/FB_MKS909.TcPOU]: Should be verbose

File: L2SIVacuum/POUs/Functions/Gauges/FB_MKS909.TcPOU [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  		GaugeDisconnected:
+			fbLogger(sMsg:='Gauge Disconnected.', eSevr:=TcEventSeverity.Critical);
```
Comment (slacAWallace) [inline L2SIVacuum/POUs/Functions/Gauges/FB_MKS909.TcPOU:119]: Maybe warning?

**Comment by ghalym** [comment]
I had them in the commits.
I will include them in the PR.

**Review by slacAWallace — CHANGES_REQUESTED** [review]
I think some of the gauge messages are a bit too severe. Otherwise looks good.

**Pull request merged**